test/customer_test.rb

```diff
@@ -19,4 +27,18 @@ def test_customer_has_a_last_name
 
     assert_equal "Ondricka", customer.last_name
   end
+
+  def test_finds_invoices_returns_all_invoices_for_associated_customer
+    result_1 = @customer_repository.customers[0].invoices
+    result_2 = @customer_repository.customers[5].invoices
+
+    assert_equal 8, result_1.count
+    assert_equal 0, result_2.count
+  end
+
+  private
+
+  def fixtures_directory
+    File.expand_path('../../data/fixtures', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.